### DISCUSSION TITLE
Automatically add credits (video URL) in .txt

### DISCUSSION THREAD

**Discussion by gt0203** [Serene-Arc/bulk-downloader-for-reddit#355] [Q&A] [answered]
Hello,

I would like to create a .txt file automatically with the URLs of the videos in my compilation.
Example : 

```
credits.txt
https://www.reddit.com/r/Unexpected/comments/mv12wm/when_mom_finds_your_gummys/
https://www.reddit.com/r/Unexpected/comments/mwt19x/shit_duck/
...
```
I know a little bit about how to automatically create .txt file and write to it with python but i have no idea how to get the url of the videos i just download in a python script.

Thanks in advance :p

**Comment by aliparlakci** [comment, marked as answer]
https://github.com/aliparlakci/bulk-downloader-for-reddit/tree/development/scripts#extract-all-successfully-downloaded-ids

We have a custom script which does something very similar to your request. It captures the reddit IDs of the downloaded posts in a file. With a bit of modification to that script, you can create URLs out of that IDs and create the credits.txt
